airflow/providers/amazon/aws/operators/rds.py

```diff
@@ -551,6 +551,106 @@ def execute(self, context: 'Context') -> str:
         return json.dumps(delete_subscription, default=str)
 
 
+class RdsCreateDbInstanceOperator(RdsBaseOperator):
+    """
+    Creates an RDS DB instance
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:RdsCreateDbInstanceOperator`
+
+    :param db_instance_identifier: The DB instance identifier, must start with a letter and
+        contain from 1 to 63 letters, numbers, or hyphens
+    :param db_instance_class: The compute and memory capacity of the DB instance, for example db.m5.large
+    :param engine: The name of the database engine to be used for this instance
+    :param rds_kwargs: Named arguments to pass to boto3 RDS client function ``create_db_instance``
+        https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/rds.html#RDS.Client.create_db_instance
+    :param aws_conn_id: The Airflow connection used for AWS credentials.
+    :param wait_for_completion:  Whether or not wait for creation of the DB instance complete. (default: True)
```

Comment:
```suggestion
    :param wait_for_completion:  Whether or not wait for creation of the DB instance to complete. (default: True)
```
Same comment for other instances as well.

Reply:
Added.